applications/CompressiblePotentialFlowApplication/custom_operations/potential_to_compressible_navier_stokes_operation.cpp

```diff
@@ -0,0 +1,117 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Marco Antonio Zuñiga Perez
+//
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "utilities/parallel_utilities.h"
+#include "potential_to_compressible_navier_stokes_operation.h"
+#include "fluid_dynamics_application_variables.h"
+#include "compressible_potential_flow_application_variables.h"
+#include "custom_processes/compute_nodal_value_process.h"
+
+namespace Kratos
+{
+
+PotentialToCompressibleNavierStokesOperation::PotentialToCompressibleNavierStokesOperation(
+    Model& rModel,
+    Parameters ModelParameters)
+        : mpModel(&rModel)
+        , mParameters(ModelParameters)
+{
+    mParameters.ValidateAndAssignDefaults(this->GetDefaultParameters());
+}
+
+Operation::Pointer PotentialToCompressibleNavierStokesOperation::Create(Model &rModel,
+    Parameters Parameters) const
+{
+    return Kratos::make_shared<PotentialToCompressibleNavierStokesOperation>(rModel, Parameters);
+}
+
+const Parameters PotentialToCompressibleNavierStokesOperation::GetDefaultParameters() const
+{
+    const Parameters default_parameters = Parameters(R"({
+        "origin_model_part"       : "",
+        "destination_model_part"  : "",
+        "reference_temperature"   : 273,
+        "compute_nodal_velocities": true
+    })");
+    return default_parameters;}
+
+void PotentialToCompressibleNavierStokesOperation::Execute()
+{
+    KRATOS_TRY;
+
+    const std::string origin_model_part_name = mParameters["origin_model_part"].GetString();
+    const std::string destination_model_part_name = mParameters["destination_model_part"].GetString();
+    const double& reference_temperature = mParameters["reference_temperature"].GetDouble();
+    const double& compute_nodal_velocities = mParameters["compute_nodal_velocities"].GetBool();
+    
+    // Saving the modelparts
+    auto& mOriginModelPart = mpModel->GetModelPart(origin_model_part_name);
+    auto& mDestinationModelPart = mpModel->GetModelPart(destination_model_part_name);
```

Comment:
```suggestion
    auto& r_origin_model_part = mpModel->GetModelPart(origin_model_part_name);
    auto& r_destination_model_part = mpModel->GetModelPart(destination_model_part_name);
```
These are local (not member) reference variables. See our [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide).